mcs/class/System/Test/System.Security.Cryptography.X509Certificates/X509Certificate2Test.cs

```diff
@@ -41,6 +43,23 @@
 namespace MonoTests.System.Security.Cryptography.X509Certificates {
 	[TestFixture]
 	public class X509Certificate2Test {
+
+		CultureInfo oldcult;
```

Comment:
Use http://nunit.org/docs/2.4.8/setCulture.html instead

Reply:
Interesting!  That must be a new addition - all the tests in corlib are using the setup/teardown mechanic.  Just fixed it.